pipelines/batch/src/main/java/com/google/fhir/analytics/BulkExportApiClient.java

```diff
@@ -60,23 +60,22 @@ public class BulkExportApiClient {
    * @param fhirVersionEnum the fhir version of the resources to be exported
    * @return the absolute url via which the status and details of the job can be fetched
    */
-  public String triggerBulkExportJob(List<String> resourceTypes, FhirVersionEnum fhirVersionEnum)
-      throws BulkExportException {
+  public String triggerBulkExportJob(List<String> resourceTypes, FhirVersionEnum fhirVersionEnum) {
     Map<String, List<String>> headers = new HashMap<>();
     headers.put(HttpHeaders.ACCEPT, Arrays.asList("application/fhir+ndjson"));
     headers.put("Prefer", Arrays.asList("respond-async"));
     MethodOutcome methodOutcome =
         fetchUtil.performServerOperation(
             "export", fetchBulkExportParameters(fhirVersionEnum, resourceTypes), headers);
     if (!isStatusSuccessful(methodOutcome.getResponseStatusCode())) {
-      throw new BulkExportException(
+      throw new RuntimeException(
```

Comment:
nit: Consider using more specific but standard exceptions, e.g., `IllegalStateException` (ditto for all other `RuntimeException`s).

Reply:
Ok Done